examples/ACE/utils/NNIAP.jl

```diff
@@ -72,12 +130,45 @@ function learn!(nniap, ds, opt::Flux.Optimise.AbstractOptimiser, epochs, loss, w
         grads = ∇loss(nniap.nn, nniap.iap, ds, w_e, w_f)
         Flux.update!(optim, nniap.nn, grads[1])
         # Logging
-        curr_loss = loss(nniap.nn, nniap.iap, ds, w_e, w_f)
+        loss_time = @elapsed curr_loss = loss(nniap.nn, nniap.iap, ds, w_e, w_f)
+        println("loss_time", loss_time)
         push!(losses, curr_loss)
```

Comment:
To benchmark performance, use `@benchmark` over the function `learn!`, in serial and parallel, using the same parameters (e.g. number of epochs, etc). If you only measure energy training, redefine the serial loss function to only consider energies (just comment the lines related to force calculation). Please, do not commit the benchmarks to this PR.

`@elapsed` is not necessary here, please remove it.